package.json

```diff
@@ -36,7 +36,7 @@
     "babel-preset-react": "^6.5.0",
     "babel-preset-stage-0": "^6.5.0",
     "express": "^4.13.4",
-    "jest-cli": "^0.8.2",
+    "jest-cli": "^12.0.2",
```

Comment:
The existing tests didn't run for me until I bumped up Jest version. Node version 5.4.1.